alembic/versions/277aca1b810_migrate_to_postgis.py

```diff
@@ -0,0 +1,85 @@
+"""Migrate to PostGIS
+
+Revision ID: 277aca1b810
+Revises: 3a0765c9a2
+Create Date: 2016-04-23 08:01:49.059187
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '277aca1b810'
+down_revision = '3a0765c9a2'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+import geoalchemy2 as ga
+
+UPGRADE_QUERY = """
+UPDATE {table_name}
+SET
+    location = ST_SetSRID(ST_MakePoint(longitude, latitude), 4326);
+"""
+
+DOWNGRADE_QUERY = """
+UPDATE {table_name}
+SET
+    latitude = ST_Y(ST_TRANSFORM(location, 4326)),
+    longitude = ST_X(ST_TRANSFORM(location, 4326));
+"""
+
+def upgrade():
+    #CREATE EXTENSION IF NOT EXISTS postgis
```

Comment:
The migration script could run this command by itself (do I overlook something?):
```python
op.execute("CREATE EXTENSION IF NOT EXISTS postgis;");
```